libs/blocks/columns/columns.js

```diff
@@ -1,9 +1,13 @@
+import { decorateLinkAnalytics } from '../../martech/attributes.js';
```

Comment:
@vivgoodrich could you please take a look at the PR. As this is an issue on production for CC pages. We were hoping to get it reviewed and merged asap. cc @overmyheadandbody

Reply:
@overmyheadandbody You are accurate that this is updating some of the same things my PR will update and the values will be changed again.  However, since this is better than the 0 tracking currently being sent and the need is urgent for these live pages, I think it's fine to do a more minor update for now.